2_mpm88/assets/mpm88.py

```diff
@@ -181,10 +182,10 @@ def init_particles(x: ti.any_arr(field_dim=1), v: ti.any_arr(field_dim=1),
     grid_v = ti.Vector.ndarray(2, ti.f32, shape=(n_grid, n_grid))
     grid_m = ti.ndarray(ti.f32, shape=(n_grid, n_grid))
 
-    mod = ti.aot.Module(arch)
+    mod = ti.aot.Module(caps=['spirv_has_non_semantic_info'])
```

Comment:
Passing in magic strings seems a bit hacky. In the future we might want to export some enum classes via `export_lang.cpp`

Reply:
will work on it